nncf/quantization/quantize_model.py

```diff
@@ -444,6 +446,9 @@ def compress_weights(
     :type gptq: bool
     :param lora_correction: Indicates whether to use Lora Correction algorithm.
     :type lora_correction: bool
+    :param backup_precision: Defines a backup precision that should be used
+        for all the layers that are kept in higher precision than primary precision.
```

Comment:
Why not the description from API? IMO, looks more clear than `higher precision`.
`the precision used for layers that are not quantized to a primary precision, which is INT8_ASYM, by default.`